client/app/bundles/course/assessment/question/text-responses/commons/validations.ts

```diff
@@ -9,6 +9,7 @@ import getIndexAndKeyPath from '../../commons/utils';
 import { commonQuestionFieldsValidation } from '../../components/CommonQuestionFields';
 
 const MAX_ATTACHMENT_UPPER_LIMIT = 50;
+const MAX_ATTACHMENT_SIZE_UPPER_LIMIT = 1024;
```

Comment:
Actually, can you get the limit from the backend? (Same for MAX_ATTACHMENT_UPPER_LIMIT). So when loading the form, fetch these figures together.

Reply:
done